README.md

```diff
@@ -143,6 +145,8 @@ Awesome software projects sub-categorized by focus.
 - [OpenGeode](https://github.com/Geode-solutions/OpenGeode) – ![C++](media/icon/cplusplus.png) ![Python](media/icon/python.png) Representation and manipulation of geological models
 - [Webviz](https://github.com/equinor/webviz-config) – ![Python](media/icon/python.png) Webviz is a wrapper on top of Dash from Plotly which encourages making reusable data visualisation components and dashboards.
 - [Webviz-subsurface](https://github.com/equinor/webviz-subsurface) – ![Python](media/icon/python.png) Webviz-subsurface contains subsurface specific standard webviz containers, which are used as plugins in webviz-config.
+### Natural Language Processing
+- [geoVec](https://osf.io/4uyeq/wiki/home/) – ![Python](media/icon/python.png) "Word embeddings for application in geosciences: development, evaluation and examples of soil-related concepts" and an [implementation](https://github.com/JustinGOSSES/geoVec-playground).
```

Comment:
I don't see anywhere else in the file that uses quotation marks around the description, but not a big deal.

Otherwise the diff looks fine to me, regardless of the commit history :+1: